epub33/epub-a11y-eaa-mapping/index.html

```diff
@@ -53,69 +55,82 @@
 	<body>
 		<section id="abstract">
 			<p>The <a href="https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A32019L0882">European
-					Accessibility Act (EAA)</a> is an EU directive that establishes binding accessibility targets that
+					Accessibility Act (EAA)</a> is an EU directive that establishes binding accessibility requirements that
 				must be met by many different types of products and services to strengthen the rights of people with
 				disabilities to access goods and services, including ebooks, dedicated reading software, ereading
-				devices, and ecommerce.</p>
+				devices, digital rights management softwares and ecommerce.</p>
```

Comment:
The W3C style usually require Oxford comma (and, actually, it is explicitly better here)
```suggestion
				devices, digital rights management softwares, and ecommerce.</p>
```